src/custom/state/price/updater.ts

```diff
@@ -89,10 +88,11 @@ export default function FeesUpdater(): null {
 
   const sellCurrency = useCurrency(sellToken)
   const buyCurrency = useCurrency(buyToken)
-  const feesMap = useAllFees({ chainId })
+  const feesMap = useAllQuotes({ chainId })
   const quoteInfo = feesMap && sellToken ? feesMap[sellToken] : undefined
 
-  const refetchFee = useRefetchFeeCallback()
+  const refetchQuote = useRefetchQuoteCallback()
+  // TODO: Add refetchPrice
```

Comment:
why is this comment here? won't price also be fetched inside `useRefetchQuoteCallback`?